File: src/practice/luogu/p5705/Main.cpp
// https://www.luogu.com.cn/problem/P5705
#include <cstdio>
#include <iostream>
#include <string>

using namespace std;

int main() {
  string s;
  cin >> s;
  for (int i = s.length() - 1; i >= 0; i--) {
    printf("%c", s[i]);
  }
  printf("\n");
  return 0;
}

File: src/practice/luogu/p5705/Main.java
// https://www.luogu.com.cn/problem/P5705
package practice.luogu.p5705;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String s = scanner.next();
        StringBuilder sb = new StringBuilder();
        char[] chars = s.toCharArray();
        for (int i = chars.length - 1; i >= 0; i--) {
            sb.append(chars[i]);
        }
        System.out.println(sb);
        scanner.close();
    }
}
